### PR TITLE
ref(js): Follow rules of hooks for tooltip acceptance test effect

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by scttcper — APPROVED** [review]
sure

File: static/app/components/tooltip.tsx [hunk 1]
```diff
@@ -174,6 +170,10 @@ function useTooltipInAcceptanceTests({
 
   // Register and unregister tooltips from the store
   useEffect(() => {
+    if (!IS_ACCEPTANCE_TEST) {
+      return;
```
Comment (dashed) [inline static/app/components/tooltip.tsx:174]:
```suggestion
      return () => undefined;
```

**Review by dashed — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
with @dashed suggestion it looks good to me

**Comment by evanpurkhiser** [comment]
I think @JonasBa might have a better follow up than this

**Pull request closed without merging**